css/properties/place-content.json

```diff
@@ -47,6 +47,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "box-alignment-updates": {
+          "__compat": {
+            "description": "Updated value as per <a href='https://drafts.csswg.org/css-align-3/#align-justify-content'>CSS Box Alignment Module Level 3</a>; you can only specify a single value if it is value for both <code>align-content</code> and <code>justify-content</code>",
+            "support": {
+              "webview_android": {
+                "version_added": true
```

Comment:
@jpmedley  Any idea when this change happened to Chromia?

Reply:
I can't find evidence that either version is currently in Chrome.